components/Ui/UiCard.vue

```diff
@@ -6,14 +6,16 @@
       'card_description-whole-size': descriptionWholeSize,
     }"
   >
-    <div
-      v-if="image"
-      class="card__image"
-      :class="imageContain ? 'card__image_contain' : null"
-      :style="{
-        'background-image': `url(${image})`,
-      }"
-    />
+    <div v-if="image" class="card__image-container">
+      <nuxt-img
+        class="card__image"
+        :class="imageContain ? 'card__image_contain' : null"
+        format="webp"
+        loading="lazy"
+        sizes="sm:300px md:650px lg:250px"
```

Comment:
https://github.com/Qiskit/qiskit.org/assets/6276074/c6caa245-be3c-498a-a196-8819fa13a45a

Question: is this happening to you? (For the UiCard that's used on the event pages, some of them appear a little blurry to me.)